src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -115,6 +132,16 @@ widgets to offer a dramatically more accessible experience.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Testing accessibility on web:
+
+You can debug accessibility by visualizing the semantic nodes created for your web app using the following command line flag in profile and relese modes:
+
+`$ flutter run -d chrome --profile \
+ --dart-define=FLUTTER_WEB_DEBUG_SHOW_SEMANTICS=true`
+ 
+With the flag activated, youll be able to see your semantic nodes on top of the widgets and verify if the semantic elements are placed where they should be. If youfind examples like that please [file a bug report][]. 
```

Comment:
s/youll/you'll/
s/youfind/you find/

Reply:
That last sentence was confusing, but is this an accurate change?

```suggestion
With the flag activated, the semantic nodes appear on top of the widgets,
and you can verify that the semantic elements are placed where they should be.
If the semantic nodes are incorrectly placed, please [file a bug report][]. 
```